package.json

```diff
@@ -107,6 +107,7 @@
     "appium-support": "^2.12.0",
     "babel-core": "^6.26.0",
     "bluebird": "^3.5.1",
+    "copy-to-clipboard": "^3.0.8",
```

Comment:
we already have functionality that copies stuff to the clipboard. i don't remember where it is offhand (i think it's in the recorder). but i'd prefer not to install an additional package when i'm pretty sure we have a way to do it already.